contrib/hunspell/example/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 add_executable(
   sapi_minihunspell
 
-  main.cc
+  "main.cc"
```

Comment:
No need to quote those. We only need to quote when we're expanding variables.

Reply:
The consistency is certainly a point but our standard is to not use quote unless there is a variable expansion happening.

Reply:
Sure :) fixed.